agent/runner/actions/query_transform_test.go

```diff
@@ -15,79 +15,163 @@
 package actions
 
 import (
+	"fmt"
 	"testing"
 
 	"github.com/stretchr/testify/assert"
 )
 
-func TestDMLToSelect(t *testing.T) {
-	q, c := dmlToSelect(`SELECT nombre FROM tabla WHERE id = 0`)
-	assert.False(t, c)
-	assert.Equal(t, `SELECT nombre FROM tabla WHERE id = 0`, q)
-
-	q, c = dmlToSelect(`update ignore tabla set nombre = "carlos" where id = 0 limit 2`)
-	assert.True(t, c)
-	assert.Equal(t, `SELECT nombre = "carlos" FROM tabla WHERE id = 0`, q)
-
-	q, c = dmlToSelect(`update ignore tabla set nombre = "carlos" where id = 0`)
-	assert.True(t, c)
-	assert.Equal(t, `SELECT nombre = "carlos" FROM tabla WHERE id = 0`, q)
-
-	q, c = dmlToSelect(`update ignore tabla set nombre = "carlos" limit 1`)
-	assert.True(t, c)
-	assert.Equal(t, `SELECT nombre = "carlos" FROM tabla`, q)
-
-	q, c = dmlToSelect(`update tabla set nombre = "carlos" where id = 0 limit 2`)
-	assert.True(t, c)
-	assert.Equal(t, `SELECT nombre = "carlos" FROM tabla WHERE id = 0`, q)
-
-	q, c = dmlToSelect(`update tabla set nombre = "carlos" where id = 0`)
-	assert.True(t, c)
-	assert.Equal(t, `SELECT nombre = "carlos" FROM tabla WHERE id = 0`, q)
-
-	q, c = dmlToSelect(`update tabla set nombre = "carlos" limit 1`)
-	assert.True(t, c)
-	assert.Equal(t, `SELECT nombre = "carlos" FROM tabla`, q)
-
-	q, c = dmlToSelect(`delete from tabla`)
-	assert.True(t, c)
-	assert.Equal(t, `SELECT * FROM tabla`, q)
-
-	q, c = dmlToSelect(`delete from tabla join tabla2 on tabla.id = tabla2.tabla2_id`)
-	assert.True(t, c)
-	assert.Equal(t, `SELECT 1 FROM tabla join tabla2 on tabla.id = tabla2.tabla2_id`, q)
-
-	q, c = dmlToSelect(`insert into tabla (f1, f2, f3) values (1,2,3)`)
-	assert.True(t, c)
-	assert.Equal(t, `SELECT * FROM tabla  WHERE f1=1 and f2=2 and f3=3`, q)
-
-	q, c = dmlToSelect(`insert into tabla (f1, f2, f3) values (1,2)`)
-	assert.True(t, c)
-	assert.Equal(t, `SELECT * FROM tabla  LIMIT 1`, q)
-
-	q, c = dmlToSelect(`insert into tabla set f1="A1", f2="A2"`)
-	assert.True(t, c)
-	assert.Equal(t, `SELECT * FROM tabla WHERE f1="A1" AND  f2="A2"`, q)
-
-	q, c = dmlToSelect(`replace into tabla set f1="A1", f2="A2"`)
-	assert.True(t, c)
-	assert.Equal(t, `SELECT * FROM tabla WHERE f1="A1" AND  f2="A2"`, q)
-
-	q, c = dmlToSelect("insert into `tabla-1` values(12)")
-	assert.True(t, c)
-	assert.Equal(t, "SELECT * FROM `tabla-1` LIMIT 1", q)
-
-	q, c = dmlToSelect(`UPDATE
-  employees2
-SET
-  first_name = 'Joe',
-  emp_no = 10
-WHERE
-  emp_no = 3`)
-	assert.True(t, c)
-	assert.Equal(t, "SELECT first_name = 'Joe',   emp_no = 10 FROM employees2 WHERE emp_no = 3", q)
+func Test_dmlToSelect(t *testing.T) {
```

Comment:
nit: is than func name intentional or a temporary rename?

Reply:
I just wanted to have name explaining its private func. Its not super important so I can keep original one.